[i18n] Translation strings

pedro-mendonca opened the pull request:
- [x] Typo in context 'book' -> 'block'
- [x] Missing Gettext function in 'Close Field'

pedro-mendonca committed:
[i18n] Translation strings

- [x] Typo in context 'book' -> 'block'
- [x] Missing Gettext function in 'Close Field'

rheinardkorf reviewed (changes requested):
Awesome thanks! Can you please use `esc_attr_e` for the attribute?

pedro-mendonca commented:
@rheinardkorf I've changed the attribute to `esc_attr_e` and changed the PR.
I've followed the code pattern you have in https://github.com/rheinardkorf/advanced-custom-blocks/blob/develop/php/post-types/class-block-post.php#L173

pedro-mendonca committed:
Change escape function to `esc_attr_e`

rheinardkorf commented:
Thanks for picking that one up too!
